modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
+.. Standalone server configuration files
+.. `standalone_xml_history` subdirectory for version control
+.. `logging.properties` for logger configuration
+.. `mgmt-users.properties` for login credentials
+.. `mgmt-groups.properties` for login roles.
+
+. **deployments**: This directory holds:
+.. Application deployment files used by Jakarta EE, such as EAR, WAR, and JAR files.
+.. Marker files
+
+. **lib**: Contains common JAR files.
+
+* After running JBoss EAP 8 the first time, the following subdirectories of standalone are created:
++
+[subs="+quotes,+macros"]
+----
+data/
+log/
+tmp/
+----
+
+* Breif description of these subdirectories:
+. **data**: A storage location for subsystems handling file system content, like message queues, in-memory databases, or distributed transaction information.
+
+. **log**: The default location for the server log files.
+
+. **tmp**:  For temporary files, like the shared-key mechanism for authenticating local users to the server's command-line interface (CLI).
+
+
+== JBoss EAP Base Directory
+
+* Users can separate the configuration and data directories from the JBoss EAP 8 installation directory, aiding easier upgrades.
+* This allows the binaries to be installed once and run multiple times on the same machine.
+* Two command-line properties enable this:
+
+. **jboss.home.dir**: Installation directory for JBoss EAP binaries.
+. **jboss.server.base.dir**: Base directory for server configuration files.
+
+* You can override runtime properties at the command line by using the `-D` option when starting the server.
+* For instance, when starting the server with standalone.sh:
++
+[subs="+quotes,+macros"]
+----
+$ ./standalone.sh -Djboss.server.base.dir=/path/to/base/directory -Djboss.home.dir=/path/to/home/directory
+----
+
+== Lab: Run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+*Outcome*: In this lab, you should be able to run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+* Create a custom location for the server base directory using below command.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo mkdir -p /opt/standalone-running
+----
+
+* Copy the configuration, deployments, and lib directories to the new location.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/standalone/
+[vagrant@server standalone]$ sudo cp -r configuration deployments lib /opt/standalone-running
+----
+
+* Modify the standalone server port offset.
+
+. To verify that the JBoss EAP server is utilizing the new, copied configuration files, update the port offset. You can run multiple instances of EAP, by using the port offset to avoid port conflicts.
+
+. Update the new configuration file by editing the standalone.xml file to verify that JBoss EAP is using the configuration files in the new directory. Open the `/opt/standalone-running/configuration/standalone.xml` file with a text editor.
```

Comment:
You are violating the NOTE you mentioned earlier in the section - do not edit XML directly!

Reply:
removed this section